src/views/partials/doc-resource-monsters.ejs

```diff
@@ -191,5 +180,165 @@
+      <td align="left">string</td>
+    </tr>
+    <tr>
+      <td align="left">alignment</td>
+      <td align="left">
+        The temperment of a monster. How likily will it go on a rampage versus a creature that is
+        able to negotiate for its own survival
+      </td>
+      <td align="left">string</td>
+    </tr>
+    <tr>
+      <td align="left">armor_class</td>
+      <td align="left">The difficulty for a player to successfully deal damage to a monster</td>
+      <td align="left">integer</td>
+    </tr>
+    <tr>
+      <td align="left">hit_points</td>
+      <td align="left">
+        The hit points of a monster determine how much damage it is able to take before it can be
+        defeated
+      </td>
+      <td align="left">integer</td>
+    </tr>
+    <tr>
+      <td align="left">hit_dice</td>
+      <td align="left">
+        The hit die of a monster can be used to make a version of the same monster whose hit points
+        are determined by the roll of the die. For example: A monster with 2d6 would have its hit
+        points determine by rolling a 6 sided die twice.
+      </td>
+      <td align="left">string</td>
+    </tr>
+    <tr>
+      <td align="left">speed</td>
+      <td align="left">Speed for a monster determines how fast it can move per turn.</td>
+      <td align="left">object</td>
+    </tr>
+    <tr>
+      <td align="left">strength</td>
+      <td align="left">The strength of a monster. How hard a monster can hit a player</td>
+      <td align="left">integer</td>
+    </tr>
+    <tr>
+      <td align="left">dexterity</td>
+      <td align="left">The dexterity of a monster. The monster's ability for swift movement or stealth</td>
+      <td align="left">integer</td>
+    </tr>
+    <tr>
+      <td align="left">constitution</td>
+      <td align="left">The constitution of a monster. How sturdy a monster is</td>
+
+      <td align="left">integer</td>
+    </tr>
+    <tr>
+      <td align="left">intelligence</td>
+      <td align="left">The intelligence of a monster. The monster's ability to outsmart a player</td>
+      <td align="left">integer</td>
+    </tr>
+    <tr>
+      <td align="left">wisdom</td>
+      <td align="left">The wisdom of a monster. A monster's ability to ascertain the player's plan</td>
+      <td align="left">integer</td>
+    </tr>
+    <tr>
+      <td align="left">charisma</td>
+      <td align="left">The charisma of a monster. A monster's ability to charm or intimidate a player</td>
+      <td align="left">integer</td>
+    </tr>
+    <tr>
+      <td align="left">proficiencies</td>
+      <td align="left">A list of proficiencies of a monster</td>
+      <td align="left">list</td>
+
+    </tr>
+    <tr>
+      <td align="left">damage_vulnerabilities</td>
+      <td align="left">A list of damage types that a monster will take double damage from</td>
+      <td align="left">list</td>
+      
+    </tr>
+    <tr>
+      <td align="left">damage_resistances</td>
```

Comment:
Are we missing `damage_immunities`?

Reply:
whoopsie doodle. let me add that in

Reply:
I have added damage_immunites to the table